task_3/counter.h

```diff
@@ -0,0 +1,41 @@
+#include <iostream>
+#include <boost/bind.hpp>
+#include <boost/asio.hpp>
+#include <boost/thread/thread.hpp>
+#include <experimental/filesystem>
+#include <chrono>
+#include <queue>
+#include <fstream>
+using std::string;
+using namespace std::experimental::filesystem;
```

Comment:
Generally, type **using namespace ...;** in the headers is bad practice. Force yourself to type everywhere what namespace exactly you want to use or at least define namespace shortcuts.

Reply:
ok